Include/httpClient/httpClient.h

```diff
@@ -5,9 +5,9 @@
 #include <httpClient/pal.h>
 #include <httpClient/mock.h>
 #include <httpClient/trace.h>
-#include <httpClient/async.h>
-#include <httpClient/asyncQueue.h>
-#include <httpClient/asyncQueueEx.h>
+#include <async.h>
+#include <asyncQueue.h>
+#include <asyncQueueEx.h>
```

Comment:
does this header need asyncQueueEx.h?

Reply:
removing it causes lots of breaks in all samples and clients using httpclient because they use APIs from it.  

similar to xsapi's services.h, httpClient.h is the header we expect clients to include.  its not worth adding complexity on the dev integration story for this

---
In reply to: [224275410](https://github.com/Microsoft/libHttpClient/pull/307#discussion_r224275410) [](ancestors = 224275410)